### PR TITLE
added tests for parser, fix regex to support multi params, clean matchers trims

### PR DESCRIPTION
**Pull request by itayod**

### CODE REVIEW

File: projects/transloco/src/lib/transloco.directive.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ export class TranslocoDirective implements OnInit, OnDestroy, OnChanges {
 
   ngOnInit() {
     const { runtime } = this.translocoService.config;
-debugger
```
Comment (shaharkazaz) [inline projects/transloco/src/lib/transloco.directive.ts]: 🚀

Comment (itayod) [reply projects/transloco/src/lib/transloco.directive.ts]: :)

File: projects/transloco/karma.conf.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```
Comment (NetanelBasal) [inline projects/transloco/karma.conf.js]: Let's not do it the default. We need a browser for debugging. use `cross-env` and add ENV variable that tells whether we are in CI mode. Then in the package json file add:

```json
test:ci: "cross-env CI=true command"
test: "cross-env command"
```

File: projects/transloco/src/lib/tests/transloco.parser.spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {DefaultParser} from '../../public-api';
+
+describe('TranslocoParser', () => {
```
Comment (NetanelBasal) [inline projects/transloco/src/lib/tests/transloco.parser.spec.ts]: Add a spec for when the key doesn't exist.

File: projects/transloco/karma.conf.js (hunk 2 above)
Comment (shaharkazaz) [reply projects/transloco/karma.conf.js]: @NetanelBasal good idea

**Pull request merged**